yarp.gemspec

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+Gem::Specification.new do |spec|
+  spec.name = "yarp"
+  spec.version = "0.4.0"
+  spec.authors = ["Shopify"]
+  spec.email = ["ruby@shopify.com"]
+
+  spec.summary = "Yet Another Ruby Parser"
+  spec.homepage = "https://github.com/ruby/yarp"
+  spec.license = "MIT"
+
+  spec.require_paths = ["lib"]
+  spec.files = [
+    "CODE_OF_CONDUCT.md",
+    "CONTRIBUTING.md",
+    "LICENSE.md",
+    "Makefile.in",
+    "README.md",
+    "config.h.in",
+    "config.yml",
+    "configure",
+    "ext/yarp/api_node.c",
+    "include/yarp/ast.h",
+    "lib/yarp/node.rb",
+    "lib/yarp/serialize.rb",
+    "src/node.c",
+    "src/prettyprint.c",
+    "src/serialize.c",
+    "src/token_type.c",
+    "yarp.gemspec",
+  ]
+
+  spec.files += Dir.glob([
```

Comment:
Thanks for asking this question. Where there is overlap, the files listed explicitly are _generated_ files, and are not tracked in source control (and so may not exist in a particular work tree). The goal is to ensure an error if we try to package the gem without those files.

If I'm being honest, I'm not entirely comfortable using globs for a gem manifest; but files are still being added pretty often, and I didn't want to create a lot of work for folks (and, because the other files are in source control, we can generally rely on them existing).

Once the manifest stabilizes I would _prefer_ to list every file individually ([like this](https://github.com/sparklemotion/nokogiri/blob/main/nokogiri.gemspec#L55)).

Reply:
Force-pushed an update that lists every file individually.